bref

```diff
@@ -52,6 +52,104 @@ $app->command('init', function (SymfonyStyle $io) {
     ]);
 });
 
+$app->command('local [data]', function(?string $data, SymfonyStyle $io) {
```

Comment:
Could you add a comment above this line? Something that says something like "Runs the lambda locally".

Reply:
ok no problem